package.json

```diff
@@ -55,12 +64,17 @@
     },
     "homepage": "https://thumbprint.design/tokens",
     "dependencies": {
+        "@netlify/functions": "^1.0.0",
```

Comment:
What changed that makes this needed?

Reply:
i Converted the netlify function to TypeScript and this provides the types.